changelog.d/10838.misc

```diff
@@ -0,0 +1 @@
+Rename [MSC2716](https://github.com/matrix-org/matrix-doc/pull/2716) fields and event types from `chunk` to `batch` to match the `/batch_send` endpoint.
```

Comment:
@erikjohnston Assigned you because this rename is more a taste making decision vs code and I didn't have many opinions on it in https://github.com/matrix-org/matrix-doc/pull/2716#discussion_r684574497